src/simulation/m_global_parameters.fpp

```diff
@@ -20,7 +20,7 @@ module m_global_parameters
 
     use m_derived_types        !< Definitions of the derived types
 
-#ifdef MFC_OpenACC
+#ifdef MFC_OPENACC
```

Comment:
@anandrdbz this is a bug (should be `#ifdef MFC_OpenACC`)

Reply:
Yeah this is a bug, will fix it, but it's inconsequential